src/Runner.php

```diff
@@ -30,6 +30,7 @@ public function __construct(Container $container = null)
         $commands_directory = __DIR__ . '/Commands';
         $top_namespace = 'Pantheon\Terminus\Commands';
         $this->commands = $this->getCommands(['path' => $commands_directory, 'namespace' => $top_namespace,]);
+        $this->commands[] = 'Pantheon\\Terminus\\Authorizer';
```

Comment:
It's because it's not inside the Commands directory. If we wanted to include other directories (and we will for plugins), we'd merge this array with another return from getCommands.